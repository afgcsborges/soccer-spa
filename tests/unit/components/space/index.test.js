import React from 'react'
import Space from 'components/space'
import { shallow } from 'enzyme'

describe('Space component tests', () => {
    it('should match snapshot with default props', () => {
        const wrapper = shallow(<Space />)

        expect(wrapper).toMatchSnapshot()
        expect(wrapper.find('Styled(Space)').props().align).toBe('start')
        expect(wrapper.find('Styled(Space)').props().direction).toBe('horizontal')
        expect(wrapper.find('Styled(Space)').props().size).toBe('middle')
    })

    it('should match snapshot with given props', () => {
        const wrapper = shallow(<Space align="end" direction="vertical" size="large" />)

        expect(wrapper).toMatchSnapshot()
        expect(wrapper.find('Styled(Space)').props().align).toBe('end')
        expect(wrapper.find('Styled(Space)').props().direction).toBe('vertical')
        expect(wrapper.find('Styled(Space)').props().size).toBe('large')
    })
})
