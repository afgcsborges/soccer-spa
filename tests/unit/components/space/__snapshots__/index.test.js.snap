// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot with default props 1`] = `
<Styled(Space)
  align="start"
  direction="horizontal"
  size="middle"
/>
`;

exports[`should match snapshot with given props 1`] = `
<Styled(Space)
  align="end"
  direction="vertical"
  size="large"
/>
`;
